force-app/main/default/classes/NextStepFuture.cls

```diff
@@ -27,9 +27,27 @@ public with sharing class NextStepFuture {
      * Opportunity Next Step field is a standard field
      * Update the corresponding test class AsynchronousTest and test method nextStepFuture_testUpdateAllNextStep to cover test this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
+    @future
+    public static void updateNextStep() {
     //  The body of the method
-    //}
+        String nextStep = 'Meeting in the future';
+
+        List<Account> accts = [SELECT Id, Next_Step__c
+                              FROM Account LIMIT 2000];
+        for (Account acct : accts) {
+            acct.Next_Step__c = nextStep;
+        }        
+        update accts;
+
+        List<Opportunity> opps = [SELECT Id, NextStep
+                                  FROM Opportunity LIMIT 2000];
+        for (Opportunity opp : opps) {
+            opp.NextStep = nextStep;
+        }
+        System.debug(opps);
+        update opps;
+        
```

Comment:
You might find this easier to do with an inner query. 

This is the query I used for the problem. 

for(Account account : [SELECT Next_Step__c, (SELECT NextStep FROM Opportunities)
        FROM Account